### ISSUE TITLE
Enhancement suggestion to display a priorities color reminder

### ISSUE THREAD

**Issue by Albatros78** [pixeline/bugs#581]
Hello Patrick,

rien d'obligatoire, mais si cela est rapidement faisable, ajouter un truc dans le genre pour avoir un "reminder" sur les couleurs utilisées pour les tickets.

Il faudrait reprendre les couleurs configurées et faire un truc comme cela 

![image](https://user-images.githubusercontent.com/79452305/134574601-e72f0da6-ca02-46b4-9553-9ad82bab5cd4.png)

Ou en haut à l'ouverture des tickets (peut être mieux) car sur la sidebar de gauche si il y a beaucoup de projets le reminder sera tout en bas et peut être pas visible.

![image](https://user-images.githubusercontent.com/79452305/134574967-7266c08a-0b40-4be9-96d9-3eca0bdacdf0.png)

**Comment by Patriboom** [comment]
Bonne idée.

**Comment by Albatros78** [comment]
Yes... En haut je pense car visible rapidement (moins gros que sur mon image bien sûr) !!

**Comment by Patriboom** [comment]
Voici une version que vous devrier aimer.
Ce n'est pas encore parfait, car j'aimerais que ce soit en CSS, mais c'est fonctionnel.

La légende paraît lorsqu'on passe la souris au-dessus d'un bouton coloré de priorité.

**Comment by Patriboom** [comment]
Fonctionne désormais avec

- Mes tâches
- Tâches
- Tableau de bord
- Affichage d'un billet
- Modification d'un billet
